nodejs/eslint.config.js

```diff
@@ -13,7 +13,39 @@ module.exports = tseslint.config(
   ...tseslint.configs.recommended,
   {
     rules: {
-      "@typescript-eslint/naming-convention": "error",
+      "@typescript-eslint/naming-convention": [
```

Comment:
This works though I think you could have just disabled linting on that entire file.

Reply:
Oh okay. That's a better idea. Thanks.